crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -326,8 +328,9 @@ where
             .contains(&("contenteditable".into(), "false".into()))
     }
 
-    pub fn is_immutable_link(&self) -> bool {
+    pub fn is_immutable_link_or_mention(&self) -> bool {
         matches!(self.kind, ContainerNodeKind::Link(_) if self.is_immutable())
+            || matches!(self.kind, ContainerNodeKind::Mention(_))
```

Comment:
This can probably change as we will no longer have the ability to create  an immutable `ContainerNodeKind::Link`, that is now a `ContainerNodeKind::Mention`